core/overlays/graph-stage/postgresql.yaml

```diff
@@ -28,7 +28,7 @@ items:
         - ReadWriteOnce
       resources:
         requests:
-          storage: 32Gi
+          storage: 64Gi
```

Comment:
Can we do even more considering the current quota? E.g. `64 + 32 = 96Gi`?

Reply:
yes we came